src/Telegram.php

```diff
@@ -649,6 +670,19 @@ public function isDbEnabled()
         }
     }
 
+    /**
+     * Set commands path to given array
+     *
+     * @param array $paths  Custom commands paths to set
+     * @return \Longman\TelegramBot\Telegram
+     */
+    public function setCommandsPath(array $paths)
+    {
+        $this->commands_paths = $paths;
```

Comment:
@imami Best to set as an empty array and then use the existing `addCommandsPaths` method to add paths. Just to ensure the reuse of code.